README.md
iceblacktea

changelog.txt
[iceBlackTea_1.0.0.1 2013-06-21 张春生]
1、新功能：初始化项目
2、新功能：新建dev分支
3、新功能：搭建框架

[iceBlackTea_1.0.0.1 2013-06-24 张春生]
1、新功能：增加数据库连接功能、session功能
2、新功能：增加英雄数值计算页面
3、新功能：增加登录功能

[iceBlackTea_1.0.0.1 2013-06-25 张春生]
1、新功能：增加首页页面
2、新功能：增加英雄属性维护功能

[iceBlackTea_1.0.0.1 2013-06-28 张春生]
1、新功能：增加战斗模拟页面
2、新功能：增加socket连接

[iceBlackTea_1.0.0.1 2013-07-03 张春生]
1、新功能：增加战斗系统

[iceBlackTea_1.0.0.1 2013-07-05 张春生]
1、新功能：增加服务器列表管理功能

[iceBlackTea_1.0.0.1 2013-07-05 张春生]
1、新功能：增加任务管理功能

[iceBlackTea_1.0.0.1 2013-07-12 张春生]
1、新功能：增加怪物管理功能
2、新功能：增加副本怪物组管理功能
3、新功能：增加副本管理功能

[iceBlackTea_1.0.0.1 2013-07-13 张春生]
1、新功能：任务管理增加导出任务功能
2、新功能：socket连接增加获得副本信息、获得副本怪物组信息功能
3、新功能：新增上传文件功能

[iceBlackTea_1.0.0.1 2013-07-15 张春生]
1、新功能：增加伙伴管理功能
2、新功能：增加昵称导出功能

[iceBlackTea_1.0.0.1 2013-07-18 张春生]
1、新功能：增加武器装备管理

[iceBlackTea_1.0.0.1 2013-07-19 张春生]
1、新功能：增加城市管理功能
2、新功能：增加NPC管理功能

[iceBlackTea_1.0.0.1 2013-07-20 张春生]
1、新功能：增加角色装备功能
2、新功能：增加背包添加武器、删除武器功能

[iceBlackTea_1.0.0.1 2013-07-22 张春生]
1、新功能：增加副本事件管理

[iceBlackTea_1.0.0.1 2013-07-23 张春生]
1、新功能：调整副本结构

[iceBlackTea_1.0.0.1 2013-07-27 张春生]
1、新功能：增加离开副本功能
2、新功能：增加获得伙伴功能
3、新功能：增加添加物品至背包功能
4、新功能：增加触发副本事件功能

[iceBlackTea_1.0.0.1 2013-07-31 张春生]
1、新功能：增加改变阵型方法

[iceBlackTea_1.0.0.1 2013-08-02 张春生]
1、新功能：增加聊天功能

[iceBlackTea_1.0.0.1 2013-08-04 张春生]
1、新功能：增加更改阵型功能
2、新功能：增加购买物品功能
3、新功能：增加获得、添加、删除好友功能

[iceBlackTea_1.0.0.1 2013-08-06 张春生]
1、新功能：增加竞技场功能
2、新功能：增加项目说明

[iceBlackTea_1.0.0.1 2013-08-07 张春生]
1、新功能：整理项目代码

[iceBlackTea_1.0.0.1 2013-08-09 张春生]
1、新功能：新增接任务功能

[iceBlackTea_1.0.0.1 2013-08-13 张春生]
1、新功能：新增交任务功能
2、新功能：增加任务完成判断功能

[iceBlackTea_1.0.0.1 2013-08-21 张春生]
1、新功能：调整菜单系统

[iceBlackTea_1.0.0.1 2013-08-22 张春生]
1、新功能：调整角色管理页面
2、新功能：增加读取redis功能

[iceBlackTea_1.0.0.1 2013-08-23 张春生]
1、新功能：调整角色管理页面
2、新功能：增加角色管理页面
3、新功能：增加装备升级功能

[iceBlackTea_1.0.0.1 2013-08-26 张春生]
1、新功能：重构项目代码

[iceBlackTea_1.0.0.1 2013-08-28 张春生]
1、新功能：增加商店管理功能

[iceBlackTea_1.0.0.1 2013-08-29 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-08-30 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-08-31 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-09-01 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-09-04 张春生]
1、重构：修改连接服务器功能
2、新功能：新增导出json对象功能

[iceBlackTea_1.0.0.1 2013-09-05 张春生]
1、重构：修改连接服务器功能

[iceBlackTea_1.0.0.1 2013-09-06 张春生]
1、重构：增加常量配置

[iceBlackTea_1.0.0.1 2013-09-09 张春生]
1、重构：测试重新连接

[iceBlackTea_1.0.0.1 2013-09-12 张春生]
1、重构：增加外网服务器ip地址

[iceBlackTea_1.0.0.1 2013-09-13 张春生]
1、Bug：修改数据库连接未释放bug

[iceBlackTea_1.0.0.1 2013-09-14 张春生]
1、重构：调整测试代码

[iceBlackTea_1.0.0.1 2013-09-14 张春生]
1、重构：调整连接池连接数

[iceBlackTea_1.0.0.1 2013-09-18 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-09-20 张春生]
1、新功能：增加技能管理功能
2、新功能：调整任务管理功能
3、新功能：调整武器装备、道具管理功能
4、新功能：增加登录功能

[iceBlackTea_1.0.0.1 2013-09-21 张春生]
1、新功能：调整角色信息

[iceBlackTea_1.0.0.1 2013-09-28 张春生]
1、新功能：调整怪物数据

[iceBlackTea_1.0.0.1 2013-10-07 张春生]
1、新功能：调整技能属性值

[iceBlackTea_1.0.0.1 2013-10-10 张春生]
1、新功能：增加导入昵称到redis功能

[iceBlackTea_1.0.0.1 2013-10-12 张春生]
1、新功能：增加场景管理功能

[iceBlackTea_1.0.0.1 2013-10-15 张春生]
1、新功能：调整武器装备数据

[iceBlackTea_1.0.0.1 2013-10-16 张春生]
1、新功能：增加日期处理功能

[iceBlackTea_1.0.0.1 2013-11-13 张春生]
1、新功能：增加heroV2管理功能

[iceBlackTea_1.0.0.1 2013-11-14 张春生]
1、新功能：增加模拟战斗功能功能

[iceBlackTea_1.0.0.1 2013-11-18 张春生]
1、新功能：增加技能功能

[iceBlackTea_1.0.0.1 2013-11-21 张春生]
1、新功能：增加技能管理功能

[iceBlackTea_1.0.0.1 2013-11-27 张春生]
1、新功能：增加额外护甲技能

[iceBlackTea_1.0.0.1 2013-11-28 张春生]
1、新功能：增加提升生命上限技能

[iceBlackTea_1.0.0.1 2013-11-30 张春生]
1、新功能：增加提升攻击力技能

[iceBlackTea_1.0.0.1 2013-12-01 张春生]
1、新功能：增加破甲加成技能

[iceBlackTea_1.0.0.1 2013-12-03 张春生]
1、新功能：增加施毒技能
2、新功能：增加攻击吸血技能

[iceBlackTea_1.0.0.1 2013-12-04 张春生]
1、新功能：增加攻击力降低幸运提高技能
2、新功能：增加额外目标伤害技能

[iceBlackTea_1.0.0.1 2013-12-05 张春生]
1、新功能：增加恢复生命技能

[iceBlackTea_1.0.0.1 2013-12-06 张春生]
1、新功能：增加提升生命技能
2、新功能：增加闪避技能
3、新功能：增加冰冻技能
4、新功能：增加沉默技能

[iceBlackTea_1.0.0.1 2013-12-08 张春生]
1、新功能：增加阵法管理

[iceBlackTea_1.0.0.1 2013-12-10 张春生]
1、新功能：增加命魂管理

[iceBlackTea_1.0.0.1 2013-12-10 张春生]
1、新功能：调整英雄数据

[iceBlackTea_1.0.0.1 2013-12-12 张春生]
1、新功能：增加武器装备数据
2、新功能：增加资质数据
3、新功能：调整命魂数据

[iceBlackTea_1.0.0.1 2013-12-16 张春生]
1、新功能：调整资质数据
2、新功能：调整命魂数据

[iceBlackTea_1.0.0.1 2013-12-18 张春生]
1、新功能：调整打造数据
2、新功能：调整宝石数据
3、新功能：调整英雄数据

[iceBlackTea_1.0.0.1 2013-12-19 张春生]
1、新功能：调整打造数据
2、新功能：调整宝石数据
3、新功能：调整技能数据

[iceBlackTea_1.0.0.1 2013-12-20 张春生]
1、新功能：调整道具数据
2、新功能：调整技能数据

[iceBlackTea_1.0.0.1 2013-12-23 张春生]
1、新功能：更新技能数据

[iceBlackTea_1.0.0.1 2013-12-26 张春生]
1、新功能：更新宝石数据

[iceBlackTea_1.0.0.1 2013-12-30 张春生]
1、新功能：更新英雄、NPC、城市数据

[iceBlackTea_1.0.0.1 2013-12-31 张春生]
1、新功能：更新英雄、NPC、城市数据

[iceBlackTea_1.0.0.1 2013-01-06 张春生]
1、新功能：更新英雄数据

[iceBlackTea_1.0.0.1 2013-01-08 张春生]
1、新功能：更新命魂数据

[iceBlackTea_1.0.0.1 2013-01-13 张春生]
1、新功能：增加阵型数据
2、新功能：更新道具数据
3、新功能：增加商店数据

[iceBlackTea_1.0.0.1 2013-01-14 张春生]
1、新功能：增加GM命令

[iceBlackTea_1.0.0.1 2013-01-17 张春生]
1、新功能：更新英雄数据

[iceBlackTea_1.0.0.1 2013-01-18 张春生]
1、新功能：增加阵型升级数据

[iceBlackTea_1.0.0.1 2014-02-11 张春生]
1、新功能：增加祭坛数据

[iceBlackTea_1.0.0.1 2014-02-12 张春生]
1、新功能：调整祭坛数据

[iceBlackTea_1.0.0.1 2014-02-13 张春生]
1、新功能：调整英雄数据
2、新功能：增加融合数据

[iceBlackTea_1.0.0.1 2014-02-17 张春生]
1、新功能：增加兑换数据

[iceBlackTea_1.0.0.1 2014-02-19 张春生]
1、新功能：更新道具数据

[iceBlackTea_1.0.0.1 2014-02-20 张春生]
1、新功能：调整战斗数据

[iceBlackTea_1.0.0.1 2014-02-25 张春生]
1、新功能：调整战斗逻辑

[iceBlackTea_1.0.0.1 2014-02-26 张春生]
1、新功能：调整战斗逻辑

[iceBlackTea_1.0.0.1 2014-02-27 张春生]
1、新功能：更新技能数据

[iceBlackTea_1.0.0.1 2014-03-03 张春生]
1、新功能：增加自由技能

[iceBlackTea_1.0.0.1 2014-03-05 张春生]
1、新功能：增加重置任务功能

[iceBlackTea_1.0.0.1 2014-03-08 张春生]
1、需求变更：更新道具数据
2、需求变更：更新技能数据

[iceBlackTea_1.0.0.1 2014-03-10 张春生]
1、需求变更：调整技能参数
2、需求变更：更新怪物数据
3、需求变更：更新任务数据

[iceBlackTea_1.0.0.1 2014-03-11 张春生]
1、需求变更：更新打造数据
2、新功能：增加GM命令，打造清零（已装备物品）
3、新功能：增加测试代码
4、新功能：增加任务测试代码

[iceBlackTea_1.0.0.1 2014-03-12 张春生]
1、新功能：新增初始化用户角色功能
2、需求变更：更新装备数据
3、需求变更：更新武器数据
4、需求变更：更新道具数据
5、需求变更：更新商店数据
6、需求变更：更新任务数据
7、需求变更：调整任务测试代码

[iceBlackTea_1.0.0.1 2014-03-13 张春生]
1、需求变更：打怪任务增加多只怪的处理

--备注分类
1、新功能：
2、Bug:
3、需求变更：
4、重构：
5、merge:

--格式：
[iceBlackTea_1.0.0.1 2013-06-17 张春生]
版本号 日期 提交人
版本号：产品名 版本类型 主版本 辅版本 新功能版本 缺陷版本

export/heros.js
[
	["id","name","type","baseLevel","xpNeeded","levelFillRate","accumulated_xp","hp","hpFillRate","attack","attLevelUpRate","defense","defLevelUpRate","focus","focusMaxIncrement","speedLevel","speed","speedMaxIncrement","dodge","dodgeMaxIncrement","criticalHit","critHitMaxIncrement","critDamage","critDamageMaxIncrement","block","blockMaxIncrement","counter","counterMaxIncrement"],
	["id","name","type","baseLevel","xpNeeded","levelFillRate","accumulated_xp","hp","hpFillRate","attack","attLevelUpRate","defense","defLevelUpRate","focus","focusMaxIncrement","speedLevel","speed","speedMaxIncrement","dodge","dodgeMaxIncrement","criticalHit","critHitMaxIncrement","critDamage","critDamageMaxIncrement","block","blockMaxIncrement","counter","counterMaxIncrement"],
	[1,"Hero1",1,1,50,10,0,200,7000,80,900,10,110,"1",34,"20","5000-2222",25,"0",10,"0",15,"40",55,"0",10,"0",15],
	[2,"Hero2",2,1,45,9.3,0,260,9600,52,480,25,200,"1",24,"26","3846-1851",28,"0",12,"0",10,"44",33,"0",16,"0",9],
	[3,"Moon Princess",1,1,55,10.3,0,175,5500,80,860,15,105,"1",31,"21","4761-2173",25,"0",9,"0",14,"50",35,"0",9,"0",14],
	[4,"Sun princess",1,1,44,9.9,0,160,6500,80,830,15,100,"1",29,"21","4761-2127",26,"0",9,"0",16,"50",30,"0",8,"0",13],
	[5,"Chief",2,1,47,9.5,0,250,9350,50,500,25,200,"1",23,"27","2703-1818",28,"0",11,"0",11,"40",30,"0",17,"0",10],
	[6,"Pupil",1,1,55,10.5,0,180,6500,85,800,12,100,"1",29,"22","4545-2083",26,"0",9,"0",14,"50",30,"0",9,"0",16],
	[7,"Doctor sister",1,1,60,11,0,180,6250,90,850,12,90,"1",31,"23","4347-2040",26,"0",11,"0",16,"50",30,"0",8,"0",17],
	[8,"Witch",1,1,65,11.5,0,180,6350,95,900,12,85,"1",33,"24","4166-1960",27,"0",10,"0",18,"50",40,"0",7,"0",18],
	[9,"Fountain protector",2,1,53,9.8,0,210,9200,55,550,30,220,"1",23,"28","2571-1785",28,"0",12,"0",12,"45",35,"0",18,"0",8],
	[10,"Time machine protector",2,1,55,9.7,0,230,9500,60,600,35,250,"1",20,"29","3448-1694",30,"0",13,"0",13,"40",40,"0",19,"0",7],
	[11,"Volcano spirit",1,1,70,12,0,180,5000,95,950,12,80,"1",35,"10","10000-4000",15,"0",8,"0",22,"60",30,"0",6,"0",20]
]